Specify ssh shell is required to enable dependency graph

djdefi opened the pull request:
### Why:

Adds clarification that the `ghe-dep-graph-enable` command needs to be run within the administrative SSH shell, and links to the relevant document for SSH access.

### What's being changed:

Calls out the requirement to run the `ghe-dep-graph-enable` command via SSH and links to the relevant section for setting up SSH shell access

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

djdefi committed:
specify ssh shell

Specify that the command should be run within the administrative SSH shell, and link to the relevant document for access.

djdefi commented on content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md:
```diff
@@ -41,7 +41,7 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
 
 {% data reusables.enterprise_site_admin_settings.sign-in %}
-1. In the administrative shell, enable the {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data variables.product.prodname_dependabot_short %}{% else %}security{% endif %} alerts for vulnerable dependencies on {% data variables.product.product_location_enterprise %}:
+1. In the [administrative SSH shell](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh#enabling-access-to-the-administrative-shell-via-ssh), enable the {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data variables.product.prodname_dependabot_short %}{% else %}security{% endif %} alerts for vulnerable dependencies on {% data variables.product.pr ...
```
I realize this is an in-line link which is not preferred, but it felt a bit long as a "For more information on SSH access:" type call out.

janiceilene replied:
👋 @djdefi I agree that it'd be too long to use add a `For more info` into the procedural step, but we avoid inline links to help with localization, accessibility, and ensure users always understand what they're going to click on. What if, instead of using an inline link, you added a note after the shell command on line 47. Something like:

```
{{#note}}

**Note**: For more information about enabling access to the administrative shell via SSH, see "[Enabling alerts for vulnerable dependencies on GitHub Enterprise Server](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh#enabling-access-to-the-administrative-shell-via-ssh)."

{{/note}}
```

djdefi committed:
Link to SSH doc as a note

djdefi committed:
note syntrax

djdefi committed:
remove extra space

hubwriter commented on content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md:
````diff
@@ -45,6 +45,10 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
  ``` shell
 $ ghe-dep-graph-enable
 ```
+{% note %}
+**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."
+{% endnote %}
````
The note needs an empty line after the opening tag and before the closing tag. Otherwise this change looks good, thanks.

```suggestion
{% note %}

**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."

{% endnote %}
```

djdefi committed:
Update content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md

Co-authored-by: hubwriter <54933897+hubwriter@users.noreply.github.com>

djdefi committed:
Update enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md

djdefi committed:
Update enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md

hubwriter commented on content/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server.md:
````diff
@@ -45,6 +45,12 @@ Before enabling {% if currentVersion ver_gt "enterprise-server@2.21" %}{% data v
  ``` shell
 $ ghe-dep-graph-enable
 ```
+{% note %}
+
+**Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."
+
+{% endnote %}
````
Thanks for making that change @djdefi.

I can see from [the staging deployment of this page](https://docs-538--djdefi-dep-graph-she.herokuapp.com/en/enterprise-server@2.22/admin/configuration/enabling-alerts-for-vulnerable-dependencies-on-github-enterprise-server) that, because it's within a list item, the note needs to be indented in the markdown to render correctly.

```suggestion
   {% note %}

   **Note**: For more information about enabling access to the administrative shell via SSH, see "[Accessing the administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/configuration/accessing-the-administrative-shell-ssh)."

   {% endnote %}
```

hubwriter committed:
Indenting note box in markdown.

I should have spotted this in my previous review, so I'm going to go ahead and apply this change.

hubwriter committed:
Merge branch 'main' into djdefi-dep-graph-shell

hubwriter commented:
@djdefi - Many thanks for raising this PR and helping to improve GitHub's documentation. 👍 
I'm merging this now.

hubwriter committed:
Merge branch 'main' into djdefi-dep-graph-shell
